src/components/landing/pages/history/history-card/history-card-list.tsx

```diff
@@ -1,18 +1,33 @@
-import React, {Fragment} from 'react'
+import React, {Fragment, useState} from 'react'
 import {HistoryEntriesProps} from "../history-content/history-content";
 import {HistoryCard} from "./history-card";
+import {PagerPagination} from "../../../../pagination/pager-pagination";
+import {PagerPage} from '../../../../pagination/pager-page';
+import {Row} from 'react-bootstrap';
 
 export const HistoryCardList: React.FC<HistoryEntriesProps> = ({entries, onPinClick}) => {
+
+    const [pageIndex, setPageIndex] = useState(0);
+    const [lastPageIndex, setLastPageIndex] = useState(0);
```

Comment:
should the pageIndex maybe live in the history content component and be the same for either view?

Reply:
well. good point. i'll fix this.

Reply:
done